co2eq/energyusage.yml

```diff
@@ -0,0 +1,331 @@
+_children:
+  AT:
+   heating:
+       kwh:
+         year:
+           m2: # kwh/year/m2
+             amount: 143.049
+             source: https://www.odyssee-mure.eu/publications/efficiency-by-sector/households/heating-consumption-per-m2.html
+   electricity: # cooling, lightning and electrical appliances
+     kwh:
+       year: # kwh/year
+         amount: 3023.8
+         source: https://www.eea.europa.eu/data-and-maps/indicators/progress-on-energy-efficiency-in-europe-3/assessment
+  BE:
+   electricity: # cooling, lightning and electrical appliances
+     kwh:
```

Comment:
I would put two keys here:
- `kWhPerYear`
- `kWhPerYearPerM2`
and then we can select the one needed.

Also I would add assumptions. For example, for kWhPerYear I would add a comment saying that it is computed for all households?